### ISSUE TITLE
Fly Bug on SMP

### ISSUE THREAD

**Issue by DinoMarlir** [EinfachGustaf/einfachgustaf-issues#1]
When you leave the spawn (e.g. go the the nether) you keep your fly when you don't use the spawn elytra.

**Comment by DinoMarlir** [comment]
Currently we use [cedricmkl/craftattack-spawn-elytra](https://github.com/cedricmkl/craftattack-spawn-elytra) for our spawn elytra. We should develop our own plugin and use the [WorldGuard](https://github.com/EngineHub/WorldGuard) API.

**Comment by getSono** [comment]
Yes, I‘ve got bad internet and if I want to use the elytra I got creative fly

**Comment by DinoMarlir** [comment]
@RedstoneBulider yeah, this is an known issue (not only on our Minecraft server) and in most cases you will get teleported back after your connection reestablished.
